src/common/transformations/src/transformations/common_optimizations/sdpa_scale_fusion.cpp

```diff
@@ -22,25 +22,24 @@ namespace pass {
 
 SDPAScaleFusion::SDPAScaleFusion() {
     using namespace ov::pass::pattern;
-    using namespace ov::gen_pattern;
 
-    auto q = makePattern(ov::Rank(4));
-    auto k = makePattern(ov::Rank(4));
-    auto v = makePattern(ov::Rank(4));
-    auto mask = makePattern();
-    auto sdpa_scale = makeConst({});
-    auto scale_q = makePattern("[]") | makePattern("[1]");
-    auto scale_k = makePattern("[]") | makePattern("[1]");
+    auto q = pattern::any_input(pattern::rank_equals(4));
```

Comment:
since we are using namespace ov::pass::pattern;
```suggestion
    auto q = any_input(rank_equals(4));
```

Reply:
Done